runelite-client/src/main/java/net/runelite/client/ui/ReducedFunctionalityPopup.java

```diff
@@ -0,0 +1,46 @@
+package net.runelite.client.ui;
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Dialog;
+import javax.swing.JDialog;
+import javax.swing.JLabel;
+import javax.swing.border.EmptyBorder;
+
+public class ReducedFunctionalityPopup
+{
+	public ReducedFunctionalityPopup()
+	{
+		JDialog dialog = new JDialog();
+		dialog.setTitle("Reduced Functionality Mode");
+		dialog.setModal(true);
+		dialog.setAlwaysOnTop(true);
+		dialog.setModalityType(Dialog.ModalityType.APPLICATION_MODAL);
+
+		dialog.setLayout(new BorderLayout());
+		dialog.getContentPane().setBackground(Color.DARK_GRAY);
+
+		JLabel text = new JLabel( "<html><p color='white'>" +
+				"Runelite has not yet been updated for use with the <br>" +
+				"latest OSRS update.<br>" +
+				"As a result Runelite is running in reduced functionality mode.<br>" +
+				"A new version of Runelite for use with the new game update <br>" +
+				"should be available shortly.<br>" +
+				"<br>Thank you for your patience." +
+				"</p></html>");
+		text.setBorder(new EmptyBorder(15, 15, 15, 15));
+
+		JLabel icon = new JLabel(javax.swing.UIManager.getIcon("OptionPane.informationIcon"));
+		icon.setBorder(new EmptyBorder(10, 25, 75, 25));
+
+		dialog.add(icon, BorderLayout.WEST);
+		dialog.add(text, BorderLayout.CENTER);
```

Comment:
Add a regular OK button that does the same thing as closing the dialog, it just looks more user-friendly.